### ISSUE TITLE
Crash on cleaning config files

### ISSUE THREAD

**Issue by afuechsel** [samschott/maestral#459]
**Describe the bug**
I have a running installation with two config files: 
```
~ maestral config-files

Config name  Account                Path
maestral                            /Users/andre/Library/Application Support/maestral/maestral.ini
Malte        malte@xxxx    /Users/andre/Library/Application Support/maestral/Malte.ini
Andre        andre@xxxx  /Users/andre/Library/Application Support/maestral/Andre.ini
``` 

Unfortunately, each time I run the maestral CLI without a given -c option, the default maestral.ini (see above) is automatically created. As I am not using it, I try to clean it, which causes a hard application crash. Even worse, it cannot be killed using ^C, one have to kill the process from another terminal! 

``` 
~ maestral config-files --clean
Traceback (most recent call last):
  File "Contents/Resources/Support/Python/Resources/lib/python3.9/runpy.py", line 197, in _run_module_as_main
  File "Contents/Resources/Support/Python/Resources/lib/python3.9/runpy.py", line 87, in _run_code
  File "Contents/Resources/app/maestral_cocoa/__main__.py", line 41, in <module>
  File "Contents/Resources/app/maestral_cocoa/__main__.py", line 36, in freeze_support_cli
  File "Contents/Resources/app_packages/click/core.py", line 1137, in __call__
  File "Contents/Resources/app_packages/click/core.py", line 1062, in main
  File "Contents/Resources/app_packages/click/core.py", line 1668, in invoke
  File "Contents/Resources/app_packages/click/core.py", line 1404, in invoke
  File "Contents/Resources/app_packages/click/core.py", line 763, in invoke
  File "Contents/Resources/app_packages/maestral/cli.py", line 1243, in config_files
  File "Contents/Resources/app_packages/maestral/config/user.py", line 340, in get
configparser.NoSectionError: No section: 'account'
^C^C^C^C^C[3]    3166 terminated  maestral config-files --clean
```

**To Reproduce**
`maestral config-files --clean` 

**Expected behaviour**
The unused config file will be deleted and the following should be listed: 

```
~ maestral config-files

Config name  Account                Path
Malte        malte@xxxx    /Users/andre/Library/Application Support/maestral/Malte.ini
Andre        andre@xxxx  /Users/andre/Library/Application Support/maestral/Andre.ini
``` 

**System:**

 - Maestral version: 1.4.8 (side not: it worked with the previous version 1.4.6!) 
 - Python version:  3.9.5
 - OS: MacOS 11.6
 - Desktop environment: 
 - PyQt version (for Linux GUI):

**Comment by SamSchott** [comment]
> Unfortunately, each time I run the maestral CLI without a given -c option, the default maestral.ini (see above) is automatically created.

Yes, this is currently expected behaviour. Any command which accesses config values (even indirectly) will trigger the creation of a config file if does not yet exist. A possible solution could be to create config files lazily, i.e., only when a value is changed from the default.

> Even worse, it cannot be killed using ^C, one have to kill the process from another terminal!

Yeah, I am aware of this. This has been fixed in 2d0942a7ad585788a13339e834fb9a8dcf530f01 but the fix is not yet released.

**Comment by samschott** [comment]
The fix has been out for some time now, closing the issue,
